frontend/config/map.json

```diff
@@ -2,9 +2,28 @@
   "searchMapCenter": [44.8, 6.3],
   "maximumZoomLevel": 17,
   "searchMapZoom": 10,
-  "mapCredits": "© OpenStreetMap-Contributors, SRTM | Style: © OpenTopoMap",
-  "mapClassicLayerUrl": "https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png",
-  "mapSatelliteLayerUrl": "https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}",
```

Comment:
I have mixed feelings about removing these settings (for backwards compatibility reasons) 🤷